core/src/main/res/drawable/ic_disc_full.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
This namespace declaration is now unused

Reply:
pushed with removed from both.